test/dlc_tests/sanity/test_dlc_labels.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+import boto3
+import pytest
+
+from test.test_utils import get_repository_and_tag_from_image_uri, LOGGER
+
+
+def test_dlc_major_version_label(image, region):
+    ecr_client = boto3.client("ecr", region_name=region)
+
+    image_repository, image_tag = get_repository_and_tag_from_image_uri(image)
+    response = ecr_client.batch_get_image(
+        repositoryName=image_repository,
+        imageIds=[{"imageTag": image_tag}],
+        acceptedMediaTypes=["application/vnd.docker.distribution.manifest.v1+json"],
+    )
+    if not response.get("images"):
+        raise ValueError(
```

Comment:
why is one of these a value error and the other a key error?

Reply:
The first is an error to be raised when the "images" were not obtained through the ecr client request. I thought that semantically, it would be a missing value, and hence, ValueError.

The other is a key error because values were obtained through the ecr client request, but the specific attribute we need -- imageManifest -- is missing, hence the KeyError to denote a missing dictionary key.

Please recommend any changes that would better represent the errors.

Reply:
I think they should both be key errors -- think of it this way, if you didn't use .get, you'd get key errors in both scenarios